packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -10,6 +10,16 @@ export type SolanaAPI = Connection;
 type Context = ContextType<SolanaAPI>;
 type Request<T> = T & Context;
 
+const pushTransaction = async (request: Request<MessageTypes.PushTransaction>) => {
+    const txHash = request.payload.startsWith('0x') ? request.payload.slice(2) : request.payload;
```

Comment:
I suppose this cuts the `0x` from the payload right? I would add a comment here saying that we allow the raw transaction to have `0x` prefix but it needs to be sliced for submission.